src/js/form.js

```diff
@@ -158,7 +158,7 @@ Form.prototype = {
      * @type {string}
      */
     get id() {
-        return this.view.html.id;
+        return this.view.html.attr( 'data-form-id' );
```

Comment:
This should probably be `this.view.html.dataset.formId`

Reply:
And this should have a test

Reply:
Yiiiikes! I am really not to be trusted. 🙏🙏🙏